packages/bruno-lang/v2/src/bruToJson.js

```diff
@@ -409,9 +409,34 @@ const sem = grammar.createSemantics().addAttribute('ast', {
   settings(_1, dictionary) {
     let settings = mapPairListToKeyValPair(dictionary.ast);
 
+    const parsedSettings = {};
+
+    if (settings.followRedirects !== undefined) {
+      parsedSettings.followRedirects = typeof settings.followRedirects === 'boolean' ? settings.followRedirects : settings.followRedirects === 'true';
+    }
+
+    // Parse maxRedirects as number
+    if (settings.maxRedirects !== undefined) {
+      const maxRedirects = parseInt(settings.maxRedirects, 10);
+      if (!isNaN(maxRedirects)) {
+        parsedSettings.maxRedirects = maxRedirects;
+      }
+    }
+
+    // Parse timeout as number
+    if (settings.timeout !== undefined) {
+      const timeout = parseInt(settings.timeout, 10);
+      if (!isNaN(timeout)) {
+        parsedSettings.timeout = timeout;
+      }
+    }
+
     return {
       settings: {
-        encodeUrl: typeof settings.encodeUrl === 'boolean' ? settings.encodeUrl : settings.encodeUrl === 'true'
+        encodeUrl: typeof settings.encodeUrl === 'boolean' ? settings.encodeUrl : settings.encodeUrl === 'true',
+        followRedirects: parsedSettings.followRedirects,
+        maxRedirects: parsedSettings.maxRedirects,
+        timeout: parsedSettings.timeout
```

Comment:
Please add a test in `bruno-lang` package that tests the bruToJson and jsonToBru of request files have these settings.

You can refer recently written tests for [custom-methods](https://github.com/usebruno/bruno/tree/main/packages/bruno-lang/v2/tests/custom-methods)